packages/material-ui/package.json

```diff
@@ -48,13 +47,13 @@
     "dom-helpers": "^3.2.1",
     "hoist-non-react-statics": "^3.2.1",
     "is-plain-object": "^2.0.4",
-    "jss": "^9.8.7",
-    "jss-camel-case": "^6.0.0",
+    "jss": "^10.0.0-alpha.4",
```

Comment:
This is very ambitious. I doubt it will simplify the situation. Instead, I would try to better isolate material-ui/styles from material-ui/core/styles.